CMakeLists.txt

```
@@ -151,6 +151,12 @@ endif()
 
 if(DD4HEP_USE_EDM4HEP)
   find_package(EDM4HEP REQUIRED)
+
+  set(podio_required 0.16.0)
+  find_package(podio REQUIRED ${podio_required})
```

```suggestion
  find_package(podio ${podio_required} REQUIRED)
```
And that should error by itself, so the next check isn't needed.
(https://cmake.org/cmake/help/v3.25/command/find_package.html#basic-signature [version] comes before [REQUIRED])

---

You're right. I thought it was because the versions were all the same major version (0) but that seems not to have been the case.